app/controllers/v3/routes_controller.rb

```diff
@@ -70,6 +70,7 @@ def create
     unprocessable_space! unless space
     unprocessable_domain! unless domain
     unauthorized! unless permission_queryer.can_manage_apps_in_space?(space.guid)
+    suspended! unless permission_queryer.is_space_org_active?(space.guid)
```

Comment:
Add tests to spec/request/routes_spec.rb:
- post '/routes'
- patch '/routes/:guid'
- delete '/routes/:guid'
- post '/routes/:guid/destinations'
- patch '/routes/:guid/destinations'
- patch '/routes/:guid/destinations/:destination_guid'
- delete '/routes/:guid/destinations/:destination_guid'

Reply:
In spec/request/routes_spec.rb:

- post '/routes'

- patch '/routes/:guid'

- delete '/routes/:guid'

In spec/request/route_destinations_spec.rb:

- post '/routes/:guid/destinations'

- patch '/routes/:guid/destinations'

- update '/routes/:guid/destinations/:destination_guid'

- delete '/routes/:guid/destinations/:destination_guid'

Reply:
Done